CIPx/schema.json

```diff
@@ -0,0 +1,82 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://github.com/cardano-foundation/CIPs/blob/master/CIP?/CIP?.md",
+    "type": "object",
+    "title": "CIP?",
+    "description": "Transaction metadata schema for CIP?",
+    "required": [
+        "title",
+        "content"
+    ],
+    "properties": {
+        "title": {
+            "$id": "#/properties/title",
+            "type": "string",
+            "title": "Title",
+            "description": "Title of the communication.",
+            "maxLength": 64,
+            "examples": [
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do"
+            ]
+        },
+        "content": {
+            "$id": "#/properties/content",
+            "type": "array",
+            "title": "Content",
+            "description": "Content of the communication in chunks",
+            "items": [
+              {
+                "type": "string",
+                "maxLength": 64
+              }
+            ],
+            "examples": [
+                [
+                  "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do ",
+                  "eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut e",
+                  "nim ad minim veniam, quis nostrud exercitation ullamco laboris."
+                ]
+            ]
+        },
+        "link": {
+            "$id": "#/properties/link",
+            "type": "string",
+            "title": "Link",
+            "description": "Optional link for additional details about the communication",
+            "maxLength": 64,
```

Comment:
I know 64 is a limitation imposed by the protocol, but I think many URLs won't fit in 64 characters (especially blog post links which tend to have long descriptive URLs)

Reply:
I thought leaving this within the limit is fine, given the stake pool operators are already used to watch this limit for metadata files and would just give it an additional burden on wallets adoption it, but I don't feel strongly about this.

Reply:
We decided to drop the link all together.